ios/NaverLogin.m

```diff
@@ -3,7 +3,8 @@
 
 @interface RCT_EXTERN_MODULE(NaverLogin, NSObject)
 
-RCT_EXTERN_METHOD(login:(NSDictionary)options resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+RCT_EXTERN_METHOD(initialize:(NSDictionary)options)
+RCT_EXTERN_METHOD(login:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
```

Comment:
We have initialize method now. So, we don't need option to initialize in `login`.